js/index.js

```diff
@@ -4,6 +4,7 @@
 $(document).ready(function() {
   blog.populate();
   blog.createAll();
-  $('#post').remove();
+  // $('#post').remove();
   blog.truncateArticle();
+  newArticlePreview();
```

Comment:
`Can't find variable: newArticlePreview` - I get this error when loading this page.